### PR TITLE
Added alias to person, name_and_nick display method,

### LINKED ISSUE #257: Known By Name/Nickname
We have quite a lot of people who are known by nicknames in our congregation.

Would it be possible to have an additional "Nickname" or "Known by" field for people to aid in searching.

Perhaps display in brackets after someone's real name?

### PR DESCRIPTION
**Pull request by AndyGauge**
updated directory and show person views

Fixes #257

### CODE REVIEW

File: app/models/search.rb [hunk 1]
```diff
@@ -138,6 +138,7 @@ def name!
     where!(
       "(concat(people.first_name, ' ', people.last_name) #{like} :full_name
        or (families.name #{like} :full_name)
+       or (people.alias) #{like} :full_name
```
Comment (seven1m) [inline app/models/search.rb]: @AndyGauge should this be `#{like} :first_name`?

File: app/views/people/_form_basics.haml [hunk 2]
```diff
@@ -19,6 +19,9 @@
   = form.label :suffix
   = form.select :suffix, [nil]+Setting.get(:system, :suffixes), {}, class: 'form-control'
 .form-group
+  = form.label :alias
```
Comment (seven1m) [inline app/views/people/_form_basics.haml]: Can you add `alias` to here please: https://github.com/churchio/onebody/blob/master/config/locales/en/general.yml#L100

File: db/schema.rb [hunk 3]
```diff
@@ -339,6 +339,14 @@
...
   end
 
+  create_table "members_messages", id: false, force: :cascade do |t|
+    t.integer "message_id", limit: 4, null: false
+    t.integer "member_id",  limit: 4, null: false
+  end
+
+  add_index "members_messages", ["member_id", "message_id"], name: "index_members_messages_on_member_id_and_message_id", using: :btree
+  add_index "members_messages", ["message_id", "member_id"], name: "index_members_messages_on_message_id_and_member_id", using: :btree
+
```
Comment (seven1m) [inline db/schema.rb]: @AndyGauge can you remove this part from the diff? (I think it's part of your other PR.)

**Comment by seven1m** [comment]
This looks great! Just a few little comments...

File: app/models/search.rb (hunk 1 above)
Comment (AndyGauge) [inline app/models/search.rb]: Yes, that should work the same (even if the nickname is 2 words)  Unless two people with the same first word in their 2 word nicknames exist.... I'll change it

File: db/schema.rb (hunk 3 above)
Comment (AndyGauge) [inline db/schema.rb]: that's part #592 I'll remove it.

**Comment by AndyGauge** [comment]
I used the localization 
Known By
I hope that is well received.

**Comment by seven1m** [comment]
Awesome!

**Pull request merged**